### ISSUE TITLE
SW Saga Edition Character Create Failed error.

### ISSUE THREAD

**Issue by Jagdragoon** [BahamutDragon/pcgen#1110]
Having loaded the dataset and attempted to create a new character, I get the following:
"
Character Create Failed

Error

Unable to create a new character.
The error was: ModifierValueStore did not have a default value for: ALIGNMENT
"
To my understanding I added the data set fine (dragged each folder to its appropriate location) and all, and it does successfully select the Game Mode. Just unable to figure out how to get past this issue.
Thank you for the work!

**Comment by FerretDave** [comment]
Greetings
This is a version issue ? You’re using ‘master’ branch with 6.8? Need to
checkout the 688 branch

**Comment by Jagdragoon** [comment]
Ah, alright. I will give that one a try.

**Comment by Jagdragoon** [comment]
So, it's kind of fascinating, actually. I'm running PCGen v6.09.05. with the 688 branch dataset installed I first got a similar error as before but with handedness instead of alignment, but then the second time I clicked new character it worked.
I haven't used PCGen since ye olde days, so I haven't a clue why things be fiddly and strange. But that second one off error aside, it seems to be working.

**Comment by Jagdragoon** [comment]
Okay, nope. More fiddling shows no entries for species or class or anything. Do you just not support the "unstable" version? I don't mind downgrading, I actually just figured that was what was needed to use your most up to date datapack.

**Comment by FerretDave** [comment]
Greetings,
Bahamut supports both - the 'master' branch is for 6.09, while the 688
branch is for v6.8. so you need to ensure you've checked out the right
branch for the version you're using.
In either case - you need to copy the 'gamemode' dir for 35eBahamut  from
the git repo to to the PCGen gamemode dir - if you get the handedness
error, or unable to create characters then you've mixed the wrong versions
- easily done ;-)

Cheers
Dave (just another support of, and submitter to, Bahamuts sources)

**Comment by Jagdragoon** [comment]
Okay, I have once more made doubly sure I have a fresh install of the most up to date version of PCGen *and* the Master version of this project, making sure to move each file into its matched location in the install, and I am left once more with the alignment error.

I'm *quite* confident this isn't a version issue. When I attempted to use the 6.09 install of PCGen (the same version used in all tests, to be clear, including the first) with the 688 branch, thinking perhaps I was confused as to which version I installed, that gave the handedness/blank fields issues.

But with the appropriate versions now confirmed, I still get the alignment issue.
I promise I'm not an ignorant user. I read the readme. I've moved every file save the git_attributes, git_ignore, book_list and the readme over to the install. Everything else seems to be fine. It's just with the saga edition and attempting to create a new character.

I'm tryin', man. ;.;
